zerver/tests/tests.py

```diff
@@ -1548,6 +1548,24 @@ def check_for_toggle_param(self, pattern, param):
         user_profile = get_user_profile_by_email("hamlet@zulip.com")
         self.assertEqual(getattr(user_profile, param), False)
 
+    def check_for_toggle_param_patch(self, pattern, param):
+        # type: (str, str) -> None
```

Comment:
I think this code is similar to `check_for_toggle_param`, you can create a new function with common code and call that in both `check_for_toggle_param_patch` and `check_for_toggle_param`.